src/App.js

```diff
@@ -1,25 +1,29 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import { Header } from "./components";
+import HomePage from "./pages/HomePage/HomePage";
```

Comment:
Add entry point file to pages index.js so you can reach it easly
```suggestion
import HomePage from "./pages";
```